src/tpt_reports/tpt_reports.py

```diff
@@ -0,0 +1,87 @@
+"""tpt_reports is a report generation Python library and tool.
+
+Usage:
+  tpt-reports [--log-level=LEVEL] [JSON_FILE_PATH]
```

Comment:
Why is `JSON_FILE_PATH` listed in brackets (i.e. an optional argument)?  I don't see a default value being assigned if that argument is not provided.
```suggestion
  tpt-reports [--log-level=LEVEL] JSON_FILE_PATH
```